src/modules/billing/services/transactions-service.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const chargeModuleTransactionsConnector = require('../../../lib/connectors/charge-module/transactions');
+const Transaction = require('../../../lib/models/transaction');
+const { logger } = require('../../../logger');
+
+const mapTransaction = chargeModuleTransaction => {
+  const transaction = new Transaction();
+  transaction.id = chargeModuleTransaction.id;
+  transaction.licenceNumber = chargeModuleTransaction.licenceNumber;
+  transaction.accountNumber = chargeModuleTransaction.customerReference;
+  return transaction;
```

Comment:
Is it OK to assign properties (licenceNumber, accountNumber) to a Transaction when they are not usually part of that model?
Perhaps we could keep that data separate but still linked to the transaction so they can still be matched up later
```
return {
  licenceNumber : chargeModuleTransaction.licenceNumber,
  accountNumber : chargeModuleTransaction.customerReference,
  transaction : new Transaction(chargeModuleTransaction.id)
}
```
Also I had assumed the `.id` property of each model is its ID in our local database, whereas here it is being used to hold the external ID.
Do we need to add a `chargeModuleId` field or similar to the Transaction model for this?